aws_ami/config/ami-automl.pkr.hcl

```diff
@@ -0,0 +1,51 @@
+variable "source_ami" {
+  type    = string
+  description = "Ubuntu Server 18.04 LTS (HVM), EBS General Purpose (SSD) VolumeType"
+  default = "ami-0bdf93799014acdc4"
+  # Optional: define a filter to automatically pick the latest version of an ami as source ami.
+  // source_ami_filter {
+  //   filters = {
+  //     name                = "ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*"
+  //     root-device-type    = "ebs"
+  //     virtualization-type = "hvm"
+  //   }
+  //   most_recent = true
+  //   owners      = ["099720109477"]
+  // }
+}
+
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+# source blocks configure your builder plugins; your source is then used inside
+# build blocks to create resources. A build block runs provisioners and
+# post-processors on an instance created by the source.
+source "amazon-ebs" "automl-ami" {
+  # the profile to use in the shared credentials file for AWS.
+  // profile       = "default"
+
+  ami_name      = "ami-automl-${local.timestamp}"
+  ami_description = "AMI for the AutoML benchmark project"
+
+  # uncomment following line to create a public ami, default a private ami is created
+  // ami_groups = ["all"]
+  
+  instance_type = "t2.micro"
```

Comment:
I guess this is big enough given that we're only setting up ubuntu + python and cloning the `amlb` repo.

Reply:
I haven't experienced any issues when I tried this out.